UM/Settings/InstanceContainer.py

```diff
@@ -611,8 +611,7 @@ def deserializeMetadata(cls, serialized: str, container_id: str) -> List[Dict[st
         """
 
         serialized = cls._updateSerialized(serialized)  # Update to most recent version.
-        parser = configparser.ConfigParser(interpolation = None)
-        parser.read_string(serialized)
+        parser = FastConfigParser(serialized)
```

Comment:
It's constructing the metadata because it needs to do so anyway. Then it puts the id and the type in it (and after that, whatever it got from the file) I don't see how any work is wasted.

Reply:
Right, I thought is was overwritten but now see it was appended